libs/hw-ledger-key-ring-protocol/package.json

```diff
@@ -19,17 +19,18 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@ledgerhq/hw-transport": "workspace:*",
-    "@ledgerhq/logs": "workspace:*",
     "@ledgerhq/live-env": "workspace:*",
-    "bip32": "^4.0.0",
+    "@ledgerhq/logs": "workspace:*",
     "@noble/curves": "^1.9.7",
-    "create-hmac": "^1.1.7"
+    "bip32": "^4.0.0",
+    "create-hmac": "^1.1.7",
+    "secp256k1": "link:@noble/curves/secp256k1"
```

Comment:
Is there a specific reason to not use the package directly instead of doing a link ?
I know that @gre removed some libs too by using noble directly